diff --git a/oauth2-facebook/src/Provider/Facebook.php b/oauth2-facebook/src/Provider/Facebook.php
index 9ecfb1d..cac748d 100755
--- a/oauth2-facebook/src/Provider/Facebook.php
+++ b/oauth2-facebook/src/Provider/Facebook.php
@@ -103,7 +103,7 @@ class Facebook extends AbstractProvider
         $fields = [
             'id', 'name', 'first_name', 'last_name',
             'email', 'hometown', 'picture.type(large){url,is_silhouette}',
-            'cover{source}', 'gender', 'locale', 'link', 'timezone', 'age_range'
+            'gender', 'locale', 'link', 'timezone', 'age_range'
         ];
 
         // backwards compatibility less than 2.8
